front/src/utils/__tests__/numbers.spec.ts

```diff
@@ -80,3 +80,21 @@ describe('isFloat', () => {
     expect(isFloat(NaN)).toBe(false);
   });
 });
+
+describe('isInvalidFloatNumber', () => {
+  it('should return true if the number has more decimal places than allowed', () => {
+    expect(isInvalidFloatNumber(17.12345, 1)).toBe(true);
+  });
+
+  it('should return false if the number has the same number of decimal places than allowed', () => {
+    expect(isInvalidFloatNumber(17.1, 1)).toBe(false);
+  });
+
+  it('should return false if the number has less decimal places than allowed', () => {
+    expect(isInvalidFloatNumber(17.1, 2)).toBe(false);
+  });
+
+  it('should return false if the number is NaN', () => {
+    expect(isInvalidFloatNumber(NaN, 1)).toBe(false);
+  });
```

Comment:
Just add these tests and lgtm :)

```suggestion
  it('should return false if the number is NaN', () => {
    expect(isInvalidFloatNumber(NaN, 1)).toBe(false);
  });
  
  it('should return false if the number is a float and decimal number is NaN', () => {
    expect(isInvalidFloatNumber(10.5, NaN)).toBe(false);
  });
  
  it('should return false if the number is NaN and decimal number is NaN', () => {
    expect(isInvalidFloatNumber(NaN, NaN)).toBe(false);
  });
```

Reply:
Done !